spec/controllers/admin/categories_controller_spec.rb

```diff
@@ -16,6 +16,13 @@
     assert_response :redirect, :action => 'index'
   end
 
+  it "test_new" do
```

Comment:
I'm pretty sure that would be covered in the 'edit' tests, but I can check. It wouldn't fall under the 'test_new' block, because once you click save, you're submitting to the 'edit' action